v3.0/reference/calicoctl/resources/workloadendpoint.md

```diff
@@ -35,7 +35,7 @@ metadata:
   namespace: default
   labels:
     app: frontend
-    calico/k8s_ns: default
+    calico/kns: default
```

Comment:
This should also be `projectcalico.org/namespace: default`

It should also have another label - `projectcalico.org/orchestrator: k8s` (as should the instance above)

Reply:
Oh, OK. Have added these. Hm